### DISCUSSION TITLE
Make this repo a an official fork of vue-codemod

### DISCUSSION THREAD

**Discussion by Romainpetit** [Sidnioulz/vue-sfcmod#48] [Ideas]
`vue-codemod` has many public forks that can be publicly accessed: https://github.com/vuejs/vue-codemod/forks

**Describe the solution you'd like**
This repo could be made an official fork of `vue-codemod` to be more discoverable. 
It would also be interesting to present in the readme a comparison of the current repo with alternative forks like the popular https://github.com/originjs/vue-codemod

**Comment by Sidnioulz** [comment]
Hi! In the future, please use discussions for general discussion rather than the issue log.

As I explain in the readme, this project significantly deviates from vue-codemod. The vast majority of the original code was removed or rewritten and a lot more was added. People looking for forks are likely to look for an updated version of vue-codemod, but this project supports different targets and fulfills different goals, so those people would not find vue-sfcmod useful to their workflow.

Discoverability will be tackled once I reach 1.0 and start advertising. Though, it baffles me that several third-parties with an interest in general codemodding engines have already reached out.
